torch/nn/_functions/thnn/activation.py

```diff
@@ -95,11 +98,57 @@ def backward(self, grad_output):
             self.lower,
             self.upper,
             self.train,
-            self.inplace
+            False
         )
         return grad_input
 
 
+class SELU(InplaceFunction):
+    alpha = 1.6732632423543772848170429916717
+    scale = 1.0507009873554804934193349852946
```

Comment:
would it make sense to add a comment that these two values are fixed point solutions to equations (4) and (5) in the paper (arxiv:1706.02515) for input/output mean zero and unit variance and a normalized weight vector and that their analytical expression is given in equation (14) ?